packages/nimble-blazor/Examples/Demo.Client/Demo.Client.csproj

```diff
@@ -12,8 +12,8 @@
     <Content Remove="packages.lock.json" />
```

Comment:
@atmgrifter00 @msmithNI I'm assuming we don't want to be on dotnet 8 yet (?)
But we may want to update those other two packages Microsoft.NET.Test.Sdk (16.11.0 -> 17.9.0) and coverlet.collector (1.3.0 -> 6.0.2)